serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -43,4 +39,5 @@ components:
         message:
           type: string
 ----
-[source,yaml]
+
+include::../../pages/_common-content/report-issue.adoc[]
```

Comment:
@hmanwani-rh I forgot to add, but what do you think about adding additional resources here to give more context?

```asciidoc
== Additional resources

* xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
* xref:service-orchestration/configuring-openapi-services-endpoints.adoc[Configuring the OpenAPI services endpoints]
```

Reply:
sure, we can.

Reply:
Done, added Additional resources.